docs/get-started/deployment-and-usage.md

```diff
@@ -284,11 +284,13 @@ We have introduced the following Chart parameters.
 | `image.tag` | string | `{{ site.release }}` | NFD image tag |
 | `image.pullPolicy` | string | `Always` | Image pull policy |
 | `imagePullSecrets` | list | [] | ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this PodSpec. If specified, these secrets will be passed to individual puller implementations for them to use. For example, in the case of docker, only DockerConfig type secrets are honored. [More info](https://kubernetes.io/docs/concepts/containers/images#specifying-imagepullsecrets-on-a-pod) |
+| `serviceAccount.create` | bool | true | Specifies whether a service account should be created |
+| `serviceAccount.annotations` | dict | {} | Annotations to add to the service account |
+| `serviceAccount.name` | string |  | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
+| `rbac` | dict |  | RBAC [parameteres](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) |
```

Comment:
These shouldn't need to be touced(?) Especially, `erviceAccount.*` are not valid anymore.

Reply:
Agree... I need to rebase the branch before committing.  This is unintentional noise in the diff.

Reply:
This is taken care of in the latest update.